### PR TITLE
Setup Logging

### LINKED ISSUE #66: Implement Logger
`console.log` isn't the proper way to print statements in code.
We should be using a proper logger with log levels.

### PR DESCRIPTION
**Pull request by adityavidyadharan**
# Description

Use the `electron-logger` and `chalk` libraries for logging.
It can now be used as seen in `src/db/Procedures.ts`

```ts
import log from 'electron-log';

log.error(message);
log.warn(message);
log.info(message);
log.debug(message);
log.verbose(message);
```

Logging level is set to `debug` for development and `info` for production in `main.ts`.
We can change this to an .env file if need be in the future.

## Link to the issue you are closing

Fixes #66 

## Screenshot or other testing that you have completed

Screen Shot 2022-09-22 at 12 44 36 AM

# Checklist:

- [x] ESLint passes
- [x] Prettier passes
- [x] Included comments where necessary
- [x] Updated README if needed
- [x] Verified that code runs and generates no errors/warnings

### CODE REVIEW

**Review by SeanMulloy — APPROVED** [review]

File: src/main/main.ts [hunk 1]
```diff
@@ -11,16 +11,33 @@
...
+import { resolveHtmlPath, getDataPath } from './util';
 import MenuBuilder from './menu';
 
 class AppUpdater {
   constructor() {
+    log.transports.file.resolvePath = () =>
+      path.join(getDataPath(), 'logs/main.log');
     log.transports.file.level = 'info';
+    log.transports.console.format = (message: LogMessage, _data: unknown) => {
+      const str = `[${message.date.toUTCString()}] ${`[${message.level}]`.padEnd(
```
Comment (JackFratto) [inline src/main/main.ts:28]: looks like this is following the format of file transport. 
docs format for console: '%c{h}:{i}:{s}.{ms}%c › {text}'

File: src/main/main.ts [hunk 2]
```diff
@@ -11,16 +11,33 @@
...
 import './ipc';
+import chalk from 'chalk';
 import AppDataSource from '../data-source';
-import { resolveHtmlPath } from './util';
+import { resolveHtmlPath, getDataPath } from './util';
 import MenuBuilder from './menu';
 
 class AppUpdater {
   constructor() {
+    log.transports.file.resolvePath = () =>
```
Comment (JackFratto) [inline src/main/main.ts:24]: Should we add chalk to file logs?

File: src/main/main.ts (hunk 1 above)
Comment (adityavidyadharan) [reply src/main/main.ts:28]: It can be set to whatever you want

File: src/main/main.ts (hunk 2 above)
Comment (adityavidyadharan) [reply src/main/main.ts:24]: Chalk is just color formatting for console

**Review by Ruben-agredano — APPROVED** [review]
Looks like a good way to display messages and errors.

**Review by JackFratto — APPROVED** [review]

**Review by insaan21 — APPROVED** [review]
NICE!

**Pull request merged**